src/main/webapp/js/userMapData.json

```diff
@@ -0,0 +1,826 @@
+["Universidad Nacional de San Agustín","Peru"],
+["Asian Institute of Management","Philippines"],
+["Ateneo de Manila University","Philippines"],
+["Ateneo School of Medicine & Public Health","Philippines"],
+["Batangas State University","Philippines"],
+["De La Salle University","Philippines"],
+["EKindling","Philippines"],
+["Holy Cross of Davao College","Philippines"],
+["La Salle University","Philippines"],
+["Miriam College High School","Philippines"],
+["University of Manila","Philippines"],
+["University of the Cordilleras","Philippines"],
+["University of the Philippines","Philippines"],
+["Xavier School","Philippines"],
+["Adam Mickiewicz University","Poland"],
+["Universidade do Minho","Portugal"],
+["Universidad del Este","Puerto Rico"],
+["Carnegie Mellon University","Qatar"],
+["Qatar University","Qatar"],
+["SUNY Korea","Republic of Korea"],
+["Deltacredit","Russia"],
+["National Research University","Russia"],
+["NRU-Higher School of Economics","Russia"],
+["Edinburgh Napier School of Eng. and the Built Env.","Scotland"],
+["University of St Andrews","Scotland"],
+["Bedok Green Secondary","Singapore"],
+["Embry-Riddle Aeronautical University","Singapore"],
+["Experimental Systems and Technologies Lab","Singapore"],
+["Hwa Chong Institution","Singapore"],
+["Institute of Technical Education","Singapore"],
+["International Community School","Singapore"],
+["ITE","Singapore"],
+["Methodist Girls&#39; School","Singapore"],
+["Millennia Institute","Singapore"],
+["NAFA","Singapore"],
+["Nanyang Junior College","Singapore"],
+["Nanyang Technological University","Singapore"],
+["National University of Singapore","Singapore"],
+["Ngee Ann Polytechnic","Singapore"],
+["Punggol Green Primary School","Singapore"],
+["Raffles College of Higher Education","Singapore"],
+["Raffles Junior College","Singapore"],
+["Sengkang Secondary School","Singapore"],
+["SHATEC Pte Ltd","Singapore"],
+["Silicon Straits","Singapore"],
+["Singapore Chinese Girls' School","Singapore"],
+["Singapore Institute of Technology","Singapore"],
+["Singapore Management University","Singapore"],
+["Singapore Polytechnic","Singapore"],
+["SUTD","Singapore"],
+["Temasek Polytechnic","Singapore"],
+["The Shipping Academy","Singapore"],
+["Yale-NUS","Singapore"],
+["SC Nova Gorica","Slovenia"],
+["Univerza v Ljubljani","Slovenia"],
+["Kingsmead College","South Africa"],
+["Stellenbosch University","South Africa"],
+["University of Pretoria","South Africa"],
+["University of the Western Cape","South Africa"],
+["Harvard University","South Korea"],
+["Korea University Business School","South Korea"],
+["Myeongju Elementary School","South Korea"],
+["Centro Didáctico","Spain"],
+["IE Business School","Spain"],
+["IESE Business School","Spain"],
+["Mondragon University","Spain"],
+["Santa María del Pilar","Spain"],
+["Universidad Complutense","Spain"],
+["Universidad Complutense de Madrid","Spain"],
+["Universidad de Navarra","Spain"],
+["Universidad del Pa? Vasco","Spain"],
```

Comment:
Yes, the original data is dirty due to various reasons. Clean as much as you can.